.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           yarn install --immutable --immutable-cache
       - name: Run lint
         run: |
-          yarn run nx run-many --target=lint --max-warnings=0
+          NODE_OPTIONS="--max-old-space-size=4096" yarn run nx run-many --target=lint --max-warnings=0
```

Comment:
@eeeqeee I carried over this change. Let me know if you think this change should not go into master.